src/target/adiv5.h

```diff
@@ -235,6 +235,7 @@ struct adiv5_debug_port {
 
 	/* dp_low_write returns true if no OK response, but ignores errors */
 	bool (*dp_low_write)(uint16_t addr, const uint32_t data);
+	uint32_t (*dp_low_read)(uint16_t addr);
```

Comment:
Given this addition and the proper inclusion of call-through functions in the header, might we suggest fixing the naming while here - `dp_low_write` -> `write_no_check`, `dp_low_read` -> `read_no_check`

We've been itching for a good opportunity to do this and fix `low_access` which should be `raw_access`

Reply:
good suggestion, the naming was iching me too